wiki/Article_styling_criteria/Writing/en.md

```diff
@@ -83,7 +83,7 @@ When writing out game modifiers for tournament articles, they must instead use u
 Gameplay elements must not be capitalised, unless they act as a title for a link that points to the article. The following is an example:
 
 ```markdown
-In osu!standard, beatmaps are composed of three different gameplay elements: hit circles, sliders, and spinners.
+In the osu! mode, beatmaps are composed of three different gameplay elements: hit circles, sliders, and spinners.
```

Comment:
unnecessary

Reply:
The file about ASC has been merged with #5246, the syntax should be good.